app-crypt/gpgme/gpgme-1.18.0-r1.ebuild

```diff
@@ -88,11 +88,9 @@ src_configure() {
 	# bug #847955
 	append-lfs-flags
 
-	# bug #811933 for libassuan prefix
```

Comment:
I have no idea how to setup such an environment, but I am fairly confident the solution for that issue at least now is inherently wrong. We should not be using the deprecated `libassuan-config` script which hasn't been touched in years. I think the bug report's original suggestion of setting `PKG_CONFIG_SYSROOT_DIR` would probably work, but I am not sure where the best place to do it?

Reply:
It might actually be that the libgpg-error fix is sufficient because the failed log had -I/usr/include which led to mismatched ABI errors from a guard in CBUILD's headers. So nothing to do here I think.